src/backend/resourcemanager/resourcebroker/resourcebroker_LIBYARN.c

```diff
@@ -773,7 +773,7 @@ int handleRB2RM_ClusterReport(void)
 	if ( oldGRMTotalAll.MemoryMB != PRESPOOL->GRMTotalHavingNoHAWQNode.MemoryMB ||
 		 oldGRMTotalAll.Core	 != PRESPOOL->GRMTotalHavingNoHAWQNode.Core )
 	{
-		elog(LOG, "Resource manager YARN resource broker counted YARN cluster "
+		elog(LOG, "Resource manager YARN. Resource broker counted YARN cluster "
```

Comment:
I don't think this sentence can be split in this way.  YARN resource broker is  the name, it's weird to change it. Just let it be.